explainers/subresource-loading.md

```diff
@@ -180,13 +179,20 @@ the bundle's URL.
 
 `<script type="webbundle">` doesn't support `src=` attribute. The rule must be inline.
 
-`<script type="webbundle">` doesn't fire a `load` event nor an `error` event.
+Although this explainer's title is "Subresource Loading with Web Bundles", a
+resource in a bundle can be used as `<iframe>`'s `src`, as follows:
 
-Note: We could fire `load` and `error` events if we wanted to. See [issue
-#697](https://github.com/WICG/webpackage/issues/697)
+``` html
+<script type="webbundle">
+{
+  "source": "./resources.wbn",
+  "resources": ["./iframe1.html", "./iframe2.html"]
+}
+</script>
 
-Note that neither `<script type=importmaps>` nor `<script type=speculationrules>`
-fires a `load` event for an inline rule.
+<iframe src="iframe1.html">
+<iframe src="iframe2.html">
```

Comment:
Well, currently Chromium implementation doesn't allow this (frame source must be `uuid-in-package:`).

Reply:
Okay. It would be better not to mention this here. I've reverted this addition. Thanks!